packages/composites/anchor/src/utils.ts

```diff
@@ -1,39 +1,51 @@
 import { AdapterContext, AdapterRequest, AdapterResponse } from '@chainlink/types'
-import * as TA from '@chainlink/token-allocation-adapter'
-import { BigNumber } from 'ethers'
-import { DEFAULT_TOKEN_DECIMALS } from './config'
+import { BigNumber, ethers } from 'ethers'
+import { FIXED_POINT_DECIMALS } from './config'
+import * as view from '@chainlink/terra-view-function-adapter'
 
 export const getTokenPrice = async (
   input: AdapterRequest,
   context: AdapterContext,
-  symbol: string,
-  decimals = DEFAULT_TOKEN_DECIMALS,
-): Promise<AdapterResponse> => {
-  const _config = TA.makeConfig()
-  const _execute = TA.makeExecute(_config)
-  const allocations = [
-    {
-      symbol,
-      balance: BigNumber.from(10).pow(decimals).toString(),
-      decimals,
-    },
-  ]
-  return await _execute({ id: input.id, data: { ...input.data, allocations } }, context)
+  feedAddress: string,
+  feedDecimals = 8, // Most Terra feed have 8dp
```

Comment:
By "most", is this something we should make more configurable?

Reply:
I actually don't think we need to make this configurable as both the ETH and LUNA feeds are to 8dp